src/input_action.rs

```diff
@@ -55,24 +28,26 @@ use crate::action_value::{ActionValue, ActionValueDim};
 /// #[input_action(output = Vec2, accumulation = Cumulative, consume_input = false)]
 /// struct Move;
 /// ```
+///
+/// All parameters match corresponding data in the trait.
 pub trait InputAction: Debug + Send + Sync + 'static {
     /// What type of value this action will output.
     ///
     /// - Use [`bool`] for button-like actions (e.g., `Jump`).
     /// - Use [`f32`] for single-axis actions (e.g., `Zoom`).
     /// - For multi-axis actions, like `Move`, use [`Vec2`] or [`Vec3`].
     ///
-    /// The type here will determine the type of the `value` field on events
-    /// e.g. [`Fired::value`](super::events::Fired::value),
-    /// [`Canceled::value`](super::events::Canceled::value).
+    /// This type will also be used for `value` field on events
+    /// e.g. [`Fired::value`](crate::events::Fired::value),
+    /// [`Canceled::value`](crate::events::Canceled::value).
     type Output: ActionOutput;
 
     /// Specifies whether this action should swallow any [`Input`](crate::input::Input)s
     /// bound to it or allow them to pass through to affect other actions.
     ///
     /// Inputs are consumed only if the action state is not equal to
```

Comment:
```suggestion
    /// Inputs are consumed when the action state is not equal to
```
I think when might work better here because it is expected for the action state to change in order for the consuming input behavior to occur or not. It is the behavior when it happens that is being modified.

Is this outside the bounds of the review?